components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -83,17 +84,17 @@ You can modify policy membership for any policy, but you can only adjust policy
 ## Policy Types
 
 IAM v2 distinguishes two types of policies: *Chef-managed* and *Custom*.
-*Chef-managed* policies are provided by Chef and are integral to the operation of Chef Automate. The policy statements in Chef-managed policies cannot be changed.
+Chef provides *Chef-managed* policies, which are integral to the operation of Chef Automate. The policy statements in Chef-managed policies cannot be changed.
 *Custom* policies are policies that you create for your own needs. You can add, edit, and delete policy statements in your custom policies.
 
 The membership can be modified in both types of policies.
 
 ## Role-Based Access Control
 
 A role is a named list of actions.
-This listing provides the benefits of *encapsulation*, where only the name is needed and is not encumbered by all the details after initial definition, and *reusability*, in applying the role to any statement that needs it.
+This listing provides the benefits of *encapsulation*, where only the name is needed and not burdened by all the details after initial definition, and *reusability*, in applying the role to any statement that needs it.
```

Comment:
```suggestion
This list provides the benefit of *encapsulation*,  so when included in a policy statement, only the role's name is needed and the policy is not burdened by all the details. Also, the list offers *reusability* as you can apply the role to any statement that needs it.
```

Reply:
I'm not wed to this suggestion, but that sentence is pretty confusing (bc roles) and I think could use some teasing apart.